fix: Pull latest from uos.

dantwinkler opened the pull request.

dbking77 committed:
Automatically reboot scanner if it reports an error code. (#44)

mintar committed:
Update package description

The driver supports all released TiM scanners by now.

mintar committed:
Update changelog

mintar committed:
0.0.10

mintar committed:
.travis.yml: Add fix for travis-ci/travis-ci#8048

Due to a change in Travis, the default Debian python path is now
ignored. This commit works around that issue by setting the PYTHONPATH
explicitly.

mintar committed:
Make output REP-117 compliant (invalid = +inf)

With this commit, the laser scan topic encodes invalid measurements as
+inf instead of 0.

Some background: REP-117 specifies 3 special values for ranges:

-inf = too close to measure
 nan = invalid measurement
+inf = out of range

Unfortunately, the SICK TiM protocol doesn't distinguish between these
events, and returns 0 for all three. The old behavior of the driver
(before this commit) was to propagate this 0 to the laser scan message.
Since 0 is below the value of the range_min field, the measurement was
correctly discarded.

However, for the navigation costmap2d this is still a problem, since
obstacles were only cleared when there was a valid measurement behind.
This commit makes costmap2d treat all invalid measurements as out of
range measurements, thereby clearing all obstacles in between.
This can lead to some obstacles being incorrectly cleared (when the
SICK TiM 0 actually means "invalid measurement" or "too close to
measure" instead of "out of range"), but this happens much less
frequently in practice than the problem of non-cleared obstacles.

Fixes #54.

Also see http://www.ros.org/reps/rep-0117.html .

mintar committed:
Update changelog

mintar committed:
0.0.11

mintar committed:
Fix typo

mintar committed:
Switch to xacro --inorder processing

mintar committed:
Add missing robot_state_publisher dependency

mintar committed:
Add roslaunch_add_file_check

mintar committed:
Add xacro runtime dependency

mintar committed:
mrs1000: Make output REP-117 compliant (invalid = +inf)

This is a port of e964fb4c to the MRS-1000.

spuetz committed:
sick mrs1000 driver (#55)

This commit adds SICK MRS-1000 support. The initialization of the device has to be different, due to that I have made the methods for initialization virtual and now the mrs1000 driver runs different init code. Also the support for PointCloud2 is new.

jspricke committed:
Fix memory leak (#56)

s should be deleted before returning.

mintar committed:
.travis.yml: catkin run_tests -> catkin test

catkin run_tests didn't run all tests, for example roslaunch_add_file_check.

mintar committed:
.travis.yml: Go back to "catkin run_tests"

"catkin run_tests" prints the full output, which "catkin test" doesn't.
The previous commit message was wrong: "catkin run_tests" actually runs
all tests, but it always returns 0. Adding catkin_test_results fixes
this.

mintar committed:
Update changelog

mintar committed:
0.0.12

artivis committed:
catkin test-flag around `roslaunch_add_file_check` (#59)

Eval `CATKIN_ENABLE_TESTING` prior to call `roslaunch_add_file_check` to ensure the function is defined.

jacobperron committed:
Add min/max angle and range parameters to URDF macros (#60)

PatrickHussey committed:
Fix mrs1000 frame_id parameter (#61)

If frame_id parameter was defined to something other than "laser", the node would not publish the /scan topic and the /cloud header.frame_id would always be "laser" even if a different one was defined. This fixes both issues.

mintar committed:
Update changelog

mintar committed:
0.0.13

mintar committed:
Add basic README

mintar committed:
CI: Remove jade (EOL)

mintar committed:
README: Reorganize CI tables

mintar committed:
CI: Fix "xz compressed control.tar files not supported"

6rs-machine commented:
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs

6rs-machine commented:
Found invalid ticket: REP-117.

6rs-machine commented:
Found invalid ticket: MRS-1000.

mintar committed:
sick_mrs1000: Fix missing cloud time stamp (#69)

The cloud timestamp was correctly set, but gets overwritten later.

Fixes #68.

6rs-machine commented:
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs

mintar committed:
URDF: Remove deprecated gazebo xmlns declarations

See http://gazebosim.org/tutorials/?tut=ros_urdf

6rs-machine commented:
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs

mintar committed:
TiM551/TiM571: Fix HEADER_FIELDS calculation

When the device name is not set, the number of footer fields is 5, not 6
as assumed before. This brings the total number of header + footer
fields plus the field that indicates RSSI data to 32, not 33.

Fixes #72.

mintar committed:
Fix warning

mintar committed:
Install udev rules during binary package installation

See here: https://answers.ros.org/question/133966/install-directly-to-etcudevrulesd/?answer=212041#post-id-212041

mintar committed:
Update changelog

mintar committed:
0.0.14

moriarty committed:
[package] run_depend libusb-1.0 -> libusb-1.0-dev (#75)

* [package] run_depend libusb-1.0 -> libusb-1.0-dev 

Change run_depend to libusb-1.0-dev

CMake Error: Project 'sick_tim' specifies '/usr/include/libusb-1.0' as an include dir, which is not found.

* REP-140 package.xml format="2"

See: http://www.ros.org/reps/rep-0140.html

Switch from REP-127 to REP-140 for <build_export_depend>.

* FIX: build_export_depend and build_depend

The Jenkins PR job failed.

Both build_depend and build_export_depend are needed.

* exec_depend libusb non-dev (review feedback)

mintar committed:
Update changelog

mintar committed:
0.0.15
